intro.txt

```diff
@@ -154,6 +167,40 @@ following procedure can be used:
 
 @endlatexonly
 
+@latexonly
```

Comment:
This change will also require an update to the automake scripts to change the .so file name. Standby for those